tests/unit/draggable/core.js

```diff
@@ -315,6 +315,26 @@ asyncTest( "blur behavior", function() {
 	} );
 } );
 
+asyncTest( "blur behavior - descendant of handle", function() {
+	expect( 2 );
+
+	var element = $( "#draggable2" ).draggable( { handle: "span" } ),
+
+		// The handle is a descendant, but we also want to grab a descendant of the handle
+		handle = element.find( "span em" ),
```

Comment:
> This does regress on https://bugs.jqueryui.com/ticket/10527, but that only affects IE8.

This is not entirely correct. This change also regresses #10527 on the latest version of Chrome for me, Mac - El Capitan with Chrome version 52.0.2743.116 (64-bit). 

When I create a draggable bootstrap dialog with a text area and click inside of the text area, a blur event is fired even though the text area has the focus and the handle for Draggable is set to the header of the dialog.

Only solution for me was to downgrade to jquery-ui 1.11.4 where the `_blurActiveElement` method's behavior is different.